### PR TITLE
500/Fix NaN prices on orders page

### LINKED ISSUE #500: Liquidity NaN denominator
Denominator values showing `NaN`  - likely from a `0` spread - needs investigation

image

### PR DESCRIPTION
**Pull request by alfetopito**
Fixes #500 

See related https://github.com/gnosis/dex-js/pull/117

Updated price calculation to use new `calculatePrice` function from `dex-js`

As a bonus, prices are now locale aware. Using a locale where `,` are decimal separators:

Captura de tela de 2020-03-11 13-47-50

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr706--dexreact.review.gnosisdev.com

File: src/components/OrdersWidget/OrderRow.tsx [hunk 1]
```diff
@@ -62,32 +55,21 @@ function displayTokenSymbolOrLink(token: TokenDetails): React.ReactNode | string
...
-    const priceFmt = calculatePrice(numeratorString, denominatorString)
-    const priceInverseFmt = calculatePrice(denominatorString, numeratorString)
-
-    return [priceFmt, priceInverseFmt]
-  }, [buyToken, order.priceDenominator, order.priceNumerator, sellToken])
+    const price = calculatePrice({
+      buyToken: { amount: order.priceNumerator, decimals: buyToken.decimals },
+      sellToken: { amount: order.priceDenominator, decimals: sellToken.decimals },
+    })
+    const priceInverse = new BigNumber(1).div(price)
```
Comment (W3stside) [inline src/components/OrdersWidget/OrderRow.tsx]: is price always `BN`? if not then this will throw occasionally

Comment (Velenir) [reply src/components/OrdersWidget/OrderRow.tsx]: Is price BN or BigNumber? If BN then maybe `price.toString(10)` is necessary

Comment (Velenir) [reply src/components/OrdersWidget/OrderRow.tsx]: Scratch that, it's BigNumber, all good

Comment (alfetopito) [reply src/components/OrdersWidget/OrderRow.tsx]: Price is always BigNumber

**Pull request merged**